yupana/processor/report_slice_processor.py

```diff
@@ -259,11 +259,39 @@ def _transform_os_kernel_version(self, host: dict):
 
         return host
 
+    def _transform_mtu(self, host: dict):
+        """Transform 'system_profile.network_interfaces[]['mtu'] to Integer."""
+        system_profile = host.get('system_profile', {})
+        network_interfaces = system_profile.get('network_interfaces')
+
+        if not network_interfaces:
+            return host
+
+        mtu_transformed = False
+        for index, item in enumerate(network_interfaces):
+            if 'mtu' not in item or not item['mtu']:
+                continue
+
+            if not isinstance(item['mtu'], int):
+                network_interfaces[index]['mtu'] = int(item['mtu'])
+                mtu_transformed = True
```

Comment:
2 things: We can remove the second `if` statement too and since the iterator also passes the same reference to a single item, we can reuse it and change the value in place:

```suggestion
        for nic in network_interfaces:
            if 'mtu' not in nic or not nic['mtu'] or isinstance(nic['mtu'], int):
                continue

                nic['mtu'] = int(nic['mtu'])
                mtu_transformed = True
```

Reply:
:+1:   that makes sense!